pkg/security/podsecuritypolicy/provider.go

```diff
@@ -296,6 +296,28 @@ func (s *simpleProvider) ValidatePod(pod *api.Pod) field.ErrorList {
 							"Flexvolume driver is not allowed to be used"))
 				}
 			}
+
+			if fsType == policy.CSI {
```

Comment:
Only perform the verification if the inline csi driver feature gate is enabled

Reply:
Fixed.